frontend/src/features/documents/documentDetail/DocumentDetailContainer.tsx

```diff
@@ -37,6 +53,26 @@ export const DocumentDetailContainer: React.FunctionComponent<IDocumentDetailCon
     fetch();
   }, [props.pimsDocument.mayanDocumentId, retrieveDocumentMetadata]);
 
+  useEffect(() => {
+    const fetch = async () => {
+      if (props.pimsDocument.mayanDocumentId !== undefined) {
```

Comment:
If this if statements fail, do we need error handling of some kind? it seems to me that the form would potentially become unusable without this data.

Reply:
That is correct, the data is necessary for the form to make sense. I am not sure what can be done for handling. We do not have a consistent way of re-fetching data in other parts of the application so re-load seems the way to go for the user.

Reply:
In that case a toast here might be nice, since I wouldn't expect users to check the console.